Sources/TuistGenerator/Generator/KnownAssetTagsGenerator.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import TuistGraph
+
+private let assetFormats: Set = [
```

Comment:
we may need to extend https://github.com/tuist/tuist/blob/main/Sources/TuistGraph/Models/Target.swift#L14 to be aware of those asset types such that they are treated as an individual path during rather than one per nested file during the loading process. 

Potentially can consolidate some of these such that `validFolderExtensions` includes `knownAssetFolders`

Reply:
I had to remove this set because not only these formats but only folders can contain tags.